js/bitget.js

```diff
@@ -2299,6 +2307,9 @@ module.exports = class bitget extends Exchange {
             if (isStopOrder) {
                 throw new InvalidOrder (this.id + ' createOrder() does not support stop orders on spot markets, only swap markets');
```

Comment:
hum how would users place spot stopOrders now with this if condition here?

Reply:
They can only place trigger order in the bitget, there is no SL/TP order for spot.

Reply:
@sc0Vu ok so the name is the confusing part here, we usually call `stopOrders` `triggerOrders and vice-versa